cert-provider/cert-provider.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+
+ACME_BIN="$(realpath ~/.acme.sh/acme.sh)"
```

Comment:
The path you reference here is a bit unintuitive, as it's nowhere to be found looking at the installation. Would be great to have a comment here the `acme.sh --install` is what created this directory and put things there.

Reply:
done